mobile-app/lib/models/learn/challenge_model.dart

```diff
@@ -328,6 +349,56 @@ class Blank {
   }
 }
 
+class QuizQuestion {
+  final String text;
+  final List<Answer> answers;
+  final int solution;
+
+  const QuizQuestion({
+    required this.text,
+    required this.answers,
+    required this.solution,
+  });
+
+  factory QuizQuestion.fromJson(Map<String, dynamic> data) {
+    // Quiz data has answer and distractors as separate fields rather than in a single array.
+    // We combine them into one so that the question schema
+    // is consistent with the other MCQ challenges.
+    final allAnswers = [
+      ...(data['distractors'] as List)
+          .map<Answer>((item) => Answer(answer: item, feedback: null)),
+      Answer(answer: data['answer'], feedback: null)
```

Comment:
No need to set the parameter as `null` explicitly as dart automatically knows it will be `null` if not given a value 
```suggestion
          .map<Answer>((item) => Answer(answer: item)),
      Answer(answer: data['answer'])
```